src/lib/dbHelpers.js

```diff
@@ -1,5 +1,5 @@
 // Sum all the amounts for all given currencies into an array of objects.
-const sumAllParticipants = participantAmount => participantAmount
+const sumAllParticipants = participantAmount => participantAmount.filter( a => !a.fspId.includes('DFSP'))
```

Comment:
@shashi165 can you explain why do we need this check? Seems very inflexible and also breaks current unit tests.